From 0cc996ba5dc04393530fdce906a4f1b7885d8aed Mon Sep 17 00:00:00 2001
From: Jiang Xin <zhiyou.jx@alibaba-inc.com>
Date: Wed, 2 Jun 2021 17:56:35 +0800
Subject: [PATCH 29/51] crypto: CI: install openssl@1.1 on macOS

Without providing the specific version, brew will install openssl@3
instead, and which is incompatible.

Signed-off-by: Jiang Xin <zhiyou.jx@alibaba-inc.com>
---
 ci/install-dependencies.sh | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/ci/install-dependencies.sh b/ci/install-dependencies.sh
index 107757a1fe..f03ad95719 100755
--- a/ci/install-dependencies.sh
+++ b/ci/install-dependencies.sh
@@ -53,6 +53,15 @@ macos-latest)
 		brew install "$BREW_PACKAGE"
 		brew link "$BREW_PACKAGE"
 	fi
+	brew install openssl@1.1
+	case "$jobname" in
+	osx-gcc)
+		brew install gcc@9
+		# Just in case the image is updated to contain gcc@9
+		# pre-installed but not linked.
+		brew link gcc@9
+		;;
+	esac
 	;;
 esac
 
-- 
patchwork
